engine/core/IgePathNode.ts

```diff
@@ -30,13 +28,14 @@ export class IgePathNode extends IgePoint3d {
 	 * @param {number} heuristic
 	 * @param {Object} parent
 	 * @param {string} direction
+	 * @param {number} z
 	 */
-	constructor (x: number, y: number, g: number, moveCost: number = 0, heuristic: number = 0, parent?: IgePathNode, direction: string = "") {
+	constructor (x: number, y: number, g: number, moveCost: number = 0, heuristic: number = 0, parent?: IgePathNode, direction: string = "", z: number = 0) {
```

Comment:
Since we're making breaking changes, I'm gonna introduce the z component in argument 3 after y `constructor(x, y, z <--`